src/ConfirmProvider/ConfirmProvider.test.tsx

```diff
@@ -152,7 +152,7 @@ describe("useConfirm", () => {
       fireEvent.click(getByText("Delete"));
       expect(queryByText("Remove this item?")).toBeTruthy();
       fireEvent.click(getByText("Yes"));
-      expect(queryByText("Remove this item?")).toBeTruthy();
+      expect(queryByText("Remove this item?")).toBeFalsy();
```

Comment:
I don't think this test is valid any more. Suggest removing it. It is checking that the dialog is still in the DOM even when the dialog is closed which is no longer the case.